mysql-test/suite/rdrs/include/have_rdrs.inc

```diff
@@ -0,0 +1,199 @@
+
+# Get available ports for REST server                          
+my $rest_server = new IO::Socket::INET(Listen    => 1, 
+                                    LocalAddr => 'localhost',
+                                    Proto     => 'tcp',);
+my $rest_server_port = $rest_server->sockport();
+
+my $grpc_server = new IO::Socket::INET(Listen    => 1, 
+                                    LocalAddr => 'localhost',
+                                    Proto     => 'tcp',);
+my $grpc_server_port = $grpc_server->sockport();
+ 
+$rest_server = undef;
+$grpc_server = undef;
+
+
+#----------------------------- create config file -------------------------------
+
+my $CONF = IO::File->new("$vardir/tmp/rdrs_config.json", 'w') or die;
+printf $CONF "{\n";
+printf $CONF "        \"RestServer\": {\n";
+printf $CONF "                \"RESTServerIP\": \"localhost\",\n";
+printf $CONF "                \"RESTServerPort\": $rest_server_port,\n";
+printf $CONF "                \"GRPCServerIP\": \"localhost\",\n";
+printf $CONF "                \"GRPCServerPort\": $grpc_server_port,\n";
+printf $CONF "                \"BufferSize\": 327680,\n";
+printf $CONF "                \"PreAllocatedBuffers\": 1024,\n";
+printf $CONF "                \"GOMAXPROCS\": -1\n";
+printf $CONF "        },\n";
+printf $CONF "        \"RonDBConfig\": {\n";
+printf $CONF "                \"IP\": \"localhost\",\n";
+printf $CONF "                \"Port\": $ENV{'NDB_MGMD_PORT'}\n";
+printf $CONF "        },\n";
+printf $CONF "        \"MySQLServer\": {\n";
+printf $CONF "                \"IP\": \"localhost\",\n";
+printf $CONF "                \"Port\": $ENV{MASTER_MYPORT},\n";
+printf $CONF "                \"User\": \"rondb\",\n";
+printf $CONF "                \"Password\": \"rondb\"\n";
+printf $CONF "        },\n";
+printf $CONF "        \"Security\": {\n";
+printf $CONF "                \"EnableTLS\": true,\n";
+printf $CONF "                \"RequireAndVerifyClientCert\": true,\n";
+printf $CONF "                \"CertificateFile\": \"\",\n";
+printf $CONF "                \"PrivateKeyFile\": \"\",\n";
+printf $CONF "                \"RootCACertFile\": \"\",\n";
+printf $CONF "                \"UseHopsWorksAPIKeys\": true,\n";
+printf $CONF "                \"HopsWorksAPIKeysCacheValiditySec\": 3\n";
+printf $CONF "        },\n";
+printf $CONF "        \"Log\": {\n";
+printf $CONF "                \"Level\": \"WARN\",\n";
+printf $CONF "                \"FilePath\": \"\",\n";
+printf $CONF "                \"MaxSizeMB\": 100,\n";
+printf $CONF "                \"MaxBackups\": 10,\n";
+printf $CONF "                \"MaxAge\": 30\n";
+printf $CONF "        }\n";
+printf $CONF "}\n";
+
+$CONF->close();
+
+EOF
+
+#--------------------------------------------------------------------------------
+
+--source $MYSQLTEST_VARDIR/tmp/have_rdrs_result.inc
+
+# skip if the required libs were not found
+if ($skip_test) {
```

Comment:
Not sure how this variable is set/unset

Reply:
this is initialized in file have_rdrs_result.inc